doc/basic_pv_pvc_dv.md

```diff
@@ -0,0 +1,219 @@
+# Storage
+
+Storage can be a tricky subject in kubernetes so we would like to give some basic guidance on how to implement storage for use with CDI. Storage in kubernetes is defined by Persistent Volumes ([PV](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)) which are a unit of storage. PVs define their size and other information needed to define the storage. When a user wants to use storage they define a Persistent Volume Claim ([PVC](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims)), which specifies the size of storage they want, and some other optional information. Kubernetes then tries to match an available PV to the PVC. If one is found, then the PVC is bound to the PV and the PVC becomes available. A pod can now mount the PVC using the pod spec when it starts.
+
+Cluster administrators can group PVs using storage classes. A [storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/) is a way for administrators to describe the classes of storage they offer. You can specify which storage class to use when you request a PVC, this allows you to control which storage is used.
```

Comment:
I don't know if I'd go into this level of detail (storage classes, PVs etc) in our docs; as opposed to pointing to K8s docs as part of a K8s storage overview piece type thing.  I don't think we want to be an addendum to Kubernetes documentation, just mention and point to it perhaps.